hydra-keystore-template

```diff
@@ -17,4 +17,4 @@ MINIONS_ED25519_PUB=
 MINIONS_ED25519_PRIV=$(cat <<EOF
 
 EOF
-)
\ No newline at end of file
+)
```

Comment:
@peeech @Br1ght0ne On line 5-6 is it Restic or Rustic? Comment says Rustic and template says Restic

Reply:
It's `restic`. https://restic.net